pull_request_template.md

```diff
@@ -1,16 +1,21 @@
-## Purpose
-_Describe the problem or feature in addition to a link to the issues._
+## Type
+- [ ] Feature
+- [ ] Bug
+- [ ] Update (fix something that did not work as expected)
+- [ ] Documentation
```

Comment:
Maybe we can make these rather match our changelog types? https://keepachangelog.com/en/1.1.0/

Reply:
If we're keeping the CHANGELOG up to date in the PR, then the type will be in there, so we can probably get rid of the type